djangocms_bootstrap4/contrib/bootstrap4_link/models.py

```diff
@@ -9,15 +9,15 @@
 
 from djangocms_link.models import AbstractLink
 from djangocms_icon.fields import Icon
-from djangocms_bootstrap4.constants import COLOR_STYLES
+from djangocms_bootstrap4.constants import COLOR_STYLE_CHOICES
 
-from .constants import LINK_CHOICES, LINK_SIZES
+from .constants import LINK_CHOICES, LINK_SIZE_CHOICES
 
 
 # 'link' type is added manually as it is only required for this plugin
-COLOR_STYLES = (
+COLOR_STYLE_CHOICES = (
     ('link', _('Link')),
-) + COLOR_STYLES
+) + COLOR_STYLE_CHOICES
```

Comment:
can you really do that? you're effectively overriding an import?

Reply:
yep you can do that, saw it already on several different addons, just to confirm with @czpython though before merging